pages/index.tsx

```diff
@@ -1,25 +1,65 @@
 import Head from 'next/head';
 import Image from 'next/image';
-import styles from '../styles/Home.module.css';
-import Link from 'next/link';
+import { css } from '@emotion/react';
+import Link from '../src/components/Link';
 
-export default function Home() {
+const containerStyle = css({
```

Comment:
It may be useful for you to combine these into a single styles object, so that they're grouped together and read more nicely:
```const styles  = {
  container: css({...})
}
```

Then, you can reference it with `<div css={styles.container} />`! 

I recommend doing this even with components with a single style, to reduce any thinking time around how you should organise your styles

Reply:
Done